src/types/result.ts

```diff
@@ -0,0 +1,20 @@
+
+export type ResultData = (
+  flagSet?: string,
```

Comment:
nit: should this be:
```suggestion
  flagSet?: FlagSet,
```